src/symbols/Node.tsx

```diff
@@ -38,7 +38,16 @@ export const Node: FC<NodeProps> = ({
   onClick,
   onPointerOver,
   onPointerOut,
-  onContextMenu
+  onContextMenu,
+  renderNode = ({ theme, node, nodeSize, active, opacity, animated }) => (
```

Comment:
The `defaultProps` are defined below. There are reasons I do that instead of initliazer props like this.